planet/src/main/java/com/google/planet/data/ItineraryGenerator.java

```diff
@@ -32,31 +40,59 @@
         int END = events.get(0).getOpeningHours().end();
 
         Collections.sort(events, Event.SortByOrder);
-        int [][] travelTimeGraph = getTravelTimeGraph(events);
+        int [][] travelTimeGraph = getEventsDistanceMatrix(events);
         List<ItineraryItem> items = scheduleItineraryInOrder(events, START, END, travelTimeGraph);
         return items;
     }
 
     // Function that creates a graph with the events as vertices and the travelling
     // time between the events as edges.
-    // For the prototype, all travelling times are set to 15 minutes 
-    public int[][] getTravelTimeGraph(List<Event> events) {
+    // For the MVP, real time traffic is NOT used.
+    // TODO: handle invalid addresses
+    private int[][] getEventsDistanceMatrix(List<Event> events) {
         int numOfNodes = events.size();
         int[][] travelTimeGraph = new int[numOfNodes][numOfNodes];
-        for (int i = 0; i < travelTimeGraph.length; i++) {
-            for (int j = 0; j < travelTimeGraph[i].length; j++) {
-                if (i == j) {
-                    travelTimeGraph[i][j] = 0;
-                }else{
-                    travelTimeGraph[i][j] = 15;
+
+        String GoogleApiKey = "AIzaSyDK36gDoYgOj4AlbCqh1IuaUuTlcpKF0ns";
```

Comment:
You're using the API key of Emily's project. Is that okay?

Reply:
Yeah any API key would work because it's on the server side. But i will change it to yours for consistency!

Reply:
Just wanted to make sure